contracts/externals/ens/ENS.sol

```diff
@@ -25,7 +25,7 @@
  * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 */
-pragma solidity >=0.4.24;
+pragma solidity ^0.5.0;
```

Comment:
funnily enough it works, it hasn't been updated to 0.5 because it's deployed on mainnet so it is not supposed to change anytime soon since it holds all the address-resolver information